mountinfo/mounted_linux.go

```diff
@@ -15,7 +15,7 @@ import (
 //
 // If a non-existent path is specified, an appropriate error is returned.
 // In case the caller is not interested in this particular error, it should
-// be handled separately using e.g. errors.Is(err, os.ErrNotExist).
+// be handled separately using e.g. errors.Is(err, fs.ErrNotExist).
```

Comment:
Is `os.ErrNotExist` deprecated yet? If not, it does not make sense to change this.

Reply:
It's not deprecated; but its recommended for new code to use errors.Is(err, fs.ErrNotExist).
Ref: https://pkg.go.dev/os#ErrNotExist

If it does not make sense, I can close this PR.  Thank you for taking the time to review.

Reply:
Makes sense, thanks!